connector_search_engine/models/se_binding.py

```diff
@@ -196,6 +193,15 @@ def recompute_json(self, force_export: bool = False):
                     record.state = "done"
                 record.error = ""
 
+    def _recompute_data(self):
+        self.date_recomputed = fields.Datetime.now()
+        data = self.index_id.model_serializer.serialize(self.record)
+        data["updated_on"] = fields.Datetime.to_string(self.date_recomputed)
```

Comment:
I'm wondering if this change won't break existing deployments when the json is built and validated via a pydantic model?

Reply:
Checked the code, it should not break.

Reply:
I don't think so, because the validation happens before, when the data is serialized.